src/pages/login/Login.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Google, Facebook } from '../../app/authentication/providers';
+
+export const Login = () => (
+  <div>
+    <Google
+      onUserConnected={userInfo => console.log(userInfo)}
+      SigninButton={() => <div>Google</div>}
+    />
+    <Facebook
+      onUserConnected={userInfo => console.log(userInfo)}
+      SigninButton={() => <div>Facebook</div>}
```

Comment:
function here ?

Reply:
It's an example since we don't have the real components displaying the google or facebook button